tests/modules/glimpse/chunk/nextflow.config

```diff
@@ -0,0 +1,14 @@
+process {
+    withName: glimpse_chunk {
+        params {
```

Comment:
You don't have to use params in the modules test, I mean it works, but it's just more typing :). You can just add the raw values in the `args` itself